### PR TITLE
Refactoring category

### PR DESCRIPTION
**Pull request by Delismachado**
Adjustments in: category Dao, Controller and Model.

### CODE REVIEW

**Review by abreumatheus — APPROVED** [review]

File: backend/controller/categoria_controller.py [hunk 1]
```diff
@@ -1,11 +1,8 @@
-import sys
-sys.path.append('.')
-
-from .base_controller import BaseController
+from backend.controller.base_controller import BaseController
 from backend.dao_bd.categoria_dao import CategoryDao
 
 class CategoryController(BaseController):
-    def __init__(self, type_entity):
+    def __init__(self):
```
Comment (lucasljoliveira-olist) [inline backend/controller/categoria_controller.py:5]: Faltou um typehint aqui com -> None

File: backend/dao_bd/categoria_dao.py [hunk 2]
```diff
@@ -1,46 +1,13 @@
...
 class CategoryDao(BaseDao):
-    def create(self, categoria: Category)-> list:
-        query = f""" INSERT INTO CATEGORY
-                            (NAME, DESCRIPTION)
-                            VALUES
-                            ('{categoria.name}',
-                            '{categoria.description}'); """
-        super().execute(query)
-            
+    def __init__(self):
```
Comment (lucasljoliveira-olist) [inline backend/dao_bd/categoria_dao.py]: Faltou um typehint aqui com -> None

**Review by lucasljoliveira-olist — CHANGES_REQUESTED** [review]

File: backend/controller/categoria_controller.py (hunk 1 above)
Comment (Delismachado) [reply backend/controller/categoria_controller.py:5]: adjustment done.

File: backend/dao_bd/categoria_dao.py (hunk 2 above)
Comment (Delismachado) [reply backend/dao_bd/categoria_dao.py]: adjustment done. 
thanks!

**Review by lucasljoliveira-olist — APPROVED** [review]

**Review by JotaCanutto — APPROVED** [review]

**Pull request merged**